Formula/jinja2-cli.rb

```diff
@@ -0,0 +1,30 @@
+class Jinja2Cli < Formula
+  include Language::Python::Virtualenv
+
+  desc "A CLI for the Jinja2 templating language (like a fancier 'envsubst')"
```

Comment:
Almost there ^^
```
04:38:16 Error: 1 problem in 1 formula detected
04:38:16 jinja2-cli:
04:38:16   * C: 4: col 9: Description shouldn't start with an indefinite article i.e. "A"
```

Reply:
OK @javian, I fixed the description string and added a better test case